src/GameSrc/setup.c

```diff
@@ -1165,7 +1165,7 @@ void pause_for_key(ulong wait_time) {
     waiting_for_key = false;
 }
 
-void splash_draw() {
+void splash_draw(bool show_splash) {
     int pal_file;
 
```

Comment:
We could probably just return early here, would make the logic clearer to follow than all of the added if blocks.

Reply:
Might not have pushed that change, not seeing it in the commit log. Want to double check before I merge?